pkg/commands/options/build.go

```diff
@@ -73,6 +74,8 @@ func AddBuildOptions(cmd *cobra.Command, bo *BuildOptions) {
 		"The maximum number of concurrent builds (default GOMAXPROCS)")
 	cmd.Flags().BoolVar(&bo.DisableOptimizations, "disable-optimizations", bo.DisableOptimizations,
 		"Disable optimizations when building Go code. Useful when you want to interactively debug the created container.")
+	cmd.Flags().StringVar(&bo.SBOM, "sbom", "go.version-m",
```

Comment:
Might want this to be a StringSlice to enable multiple formats.

Reply:
I kinda want to cross that bridge when we get to it, and don't think it'd be *that* breaking to change to a slice later 🤔

Reply:
Happy to change this now if you feel strongly!

Reply:
Meh, we can break it later before the release.